opentelemetry-api/src/opentelemetry/context/__init__.py

```diff
@@ -137,7 +130,6 @@ def get_current() -> Context:
     return _RUNTIME_CONTEXT.get_current()  # type:ignore
```

Comment:
I believe these `# type:ignore` can be removed since` _RUNTIME_CONTEXT` can no longer be None